### PR TITLE
core-svc-feat(master data service): add timestamp migration

### PR DESCRIPTION
**Pull request by rachadiannovansyah**
### Overview
- core-svc-feat(master data service): add timestamp migration

### Related to the ticket
https://app.clickup.com/t/860pxrrhy

## Attachment 
Result of migration up and down:
image

cc: @jabardigitalservice/jds-backend 

Evidence
project: Portal Jabar
title: add timestamp migration master data services
participants: @rachadiannovansyah @rindibudiaramdhan @ayocodingit @iqbal167

### CODE REVIEW

File: core-service/src/database/migrations/20230301034240_add_timestamps_on_masterdata_table.down.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+DROP COLUMN updated_at,
+DROP COLUMN created_at;
+
+ALTER TABLE spbe_rals
+DROP COLUMN updated_at,
+DROP COLUMN created_at;
+
+ALTER TABLE uptd_cabdins
+DROP COLUMN updated_at,
+DROP COLUMN created_at;
```
Comment (ayocodingit) [inline core-service/src/database/migrations/20230301034240_add_timestamps_on_masterdata_table.down.sql:15]: Please add new line in end line in each the files

**Review by iqbal167 — APPROVED** [review]

**Review by ayocodingit — APPROVED** [review]

File: core-service/src/database/migrations/20230301034240_add_timestamps_on_masterdata_table.down.sql (hunk 1 above)
Comment (rachadiannovansyah) [reply core-service/src/database/migrations/20230301034240_add_timestamps_on_masterdata_table.down.sql:15]: already [here](https://github.com/jabardigitalservice/portal-jabar-services/pull/398/commits/e0b4028c76f2a42ccd6692917cfb4112fe53c6fe) yaa @ayocodingit

**Review by rindibudiaramdhan — APPROVED** [review]
what tool you use to generate migrate files like this? kang @rachadiannovansyah

**Comment by rachadiannovansyah** [comment]
> what tool you use to generate migrate files like this? kang @rachadiannovansyah

I used it https://dbdiagram.io/
this tool help me to build a migration @rindibudiaramdhan

**Pull request merged**